.gitignore

```diff
@@ -57,3 +57,4 @@ sql/dummy_processor.mysql
 distmaker/distmaker.conf
 distmaker/out
 /tmp
+/.idea/
```

Comment:
I think I tried to add this one years ago & got told then I should do a local global gitignore https://docs.github.com/en/github/using-git/ignoring-files

Reply:
Thank you for the guide. The suggested change was a sign of my lack of control over .gitignore rather than a serious recommendation.